internal/lib/control_config.go

```diff
@@ -247,6 +256,38 @@ func (c *akoControlConfig) HttpRuleEnabled() bool {
 	return c.httpRuleEnabled
 }
 
+func (c *akoControlConfig) ControllerVersion() string {
+	return c.controllerVersion
+}
+
+func (c *akoControlConfig) SetControllerVersion(v string) {
+	c.controllerVersion = v
+}
+
+func initControllerVersion() string {
+	version := os.Getenv("CTRL_VERSION")
+	if version == "" {
+		return version
+	}
+
+	// Ensure that the controllerVersion is less than the supported Avi maxVersion and more than minVersion.
+	if CompareVersions(version, ">", GetAviMaxSupportedVersion()) {
+		utils.AviLog.Infof("Setting the client version to AVI Max supported version %s", GetAviMaxSupportedVersion())
+		version = GetAviMaxSupportedVersion()
```

Comment:
Currently, For controller version 22.1.3, we need to set it as 22.1.2 due to model issues.